### DISCUSSION TITLE
Ability to render textures from 1.20 server to 1.8 client

### DISCUSSION THREAD

**Discussion by Legend7832** [eaglerforge/EaglerForge#12] [Q&A]
So, currently it’s kind of impossible to see textures newer than 1.8 in an eaglercraft client (ex. Netherite sword, trident, totem of undying, etc.) With this modding api, would it technically be possible to use this API to be able to render in textures from 1.20 and be able to see all ores in eaglercraft using a texture pack?

Edit: Also, we could technically use the mapping generator from ViaVersion to implement this code, right?

**Comment by BarneyCompiler** [comment]
Technically yes, but it's a bit too advanced for the API right now. Maybe in the future. It would involve rendering certain textures added to a resource pack depending on the item name. It's a lot easier to do just to create a resource pack that does this. There are tutorials on youtube for it.

**Reply by AODGGLEADER** [reply]
This can not be done with eaglercraft, trust custom NBT Texturing methods don't exist I have tried.

**Comment by Legend7832** [comment]
https://github.com/eaglerforge/EaglerForge/issues/11

**Comment by AODGGLEADER** [comment]
this would technically be texturing based off NBT data, because the plugin you are talking about is called um via version and it allows Minecraft 1.20 blocks and items to be in older versions, but it uses some of the same blocks as those counterparts and just changes the nbt  data of the block itself so they can custom code it to work as said block, so yea we would need NBT Texturing methods and stuff like that

**Comment by Legend7832** [comment]
ill just leave this open for a few more days just to see if anyone has anything to say

**Reply by leahanderson1** [reply]
make a pr

**Comment by Legend7832** [comment]
I will be closing this issue. You are welcome to open up another issue on this idea.
